### ISSUE TITLE
ViewContent - wrong data.

### ISSUE THREAD

**Issue by Bashev** [magento/meta-for-magento2#27]
Facebook/Meta expects to see contents or content_ids (parameters) as array (json) instead of comma separated skus, as is now.
https://www.facebook.com/business/help/606577526529702?id=1205376682832142

![image](https://github.com/magento/meta-for-magento2/assets/1854269/723142d9-b49f-4eab-b782-eee20230e304)

This reflects to wrong ViewContent event tracking. In most of the cases if we have configurable products or grouped products they cannot be tracked because of that.

### Preconditions (*)

1.
2.

### Steps to reproduce (*)

1. Open random product.
2. Check Meta Pixel Helper
3. Look for Contents and Content_ids variables.

### Expected result (*)

1. content_ids
```
content_ids: ['123', '456']
```
2. contents
```
contents: [{id: '1234', quantity: 1}]
```
![image](https://github.com/magento/meta-for-magento2/assets/1854269/1c1642c1-4199-418f-8069-9a12aac3c65b)

### Actual result (*)

1. content_ids:
```
content_ids: '123, 456'
```

2. contents:
```
contents: ['123, 456', 'quantity': 1]
```

**Comment by Bashev** [comment]
@zlik can you zoom on this?

**Comment by zlik** [comment]
Fixed in [1.3.5](https://github.com/magento/meta-for-magento2/releases/tag/v1.3.5), closing the issue.
